schemas/stsci.edu/asdf/wcs/celestial_frame-1.1.0.yaml

```diff
@@ -0,0 +1,25 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "http://stsci.edu/schemas/asdf/wcs/celestial_frame-1.1.0"
```

Comment:
I might be misunderstanding it as well. The reason I bumped the version here and in other places is because these schemas now make reference to `frame-1.1.0` whereas previously they used `frame-1.0.0`. Maybe they should just reference a generic `frame` though (without a specific version) and then a version bump won't be necessary?